comp-layouts/card.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+     <meta charset="UTF-8">
+     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     <script src="https://kit.fontawesome.com/c4d0bd5372.js" crossorigin="anonymous"></script>
+     <title>cards</title>
+
+     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+     <link rel="stylesheet" href="/stylesheets/main.css" />
+     <link rel="stylesheet" href="/stylesheets/components/card.css" />
+     <link rel="stylesheet" href="/Stylesheets/abstract/color.css">
+     <title>Card</title>
```

Comment:
Two titles here, also you import you stylesheets to the main css

Reply:
yes i did changes